src/main/java/io/hops/util/exceptions/TagsError.java

```diff
@@ -13,9 +13,9 @@
  */
 package io.hops.util.exceptions;
 
-public class FeaturegroupMetadataError extends Exception{
+public class TagsError extends Exception{
```

Comment:
I think I would call it `TagError`, but it's up to you :)

Reply:
Changed it to TagError